### PR TITLE
Add hb.version() and rename hb.version to hb.version_string()

### PR DESCRIPTION
**Pull request by khaledhosny**
hb.version() returns an array of [major, minor, patch] as numbers.

### CODE REVIEW

File: hbjs.js [hunk 1]
```diff
@@ -533,7 +533,23 @@ function hbjs(Module) {
...
+  function version() {
+    var major = exports.malloc(4);
+    var minor = exports.malloc(4);
+    var patch = exports.malloc(4);
+    exports.hb_version(major, minor, patch);
+    var version = [
+      heapu32[major / 4],
+      heapu32[minor / 4],
+      heapu32[patch / 4]
+    ];
```
Comment (khaledhosny) [inline hbjs.js:545]: would an object be better?
```suggestion
    var version = {
      major: heapu32[major / 4],
      minor: heapu32[minor / 4],
      patch: heapu32[patch / 4]
    };
```

File: hbjs.js [hunk 2]
```diff
@@ -533,7 +533,23 @@ function hbjs(Module) {
     return trace;
   }
 
-  function get_version() {
+  function version() {
+    var major = exports.malloc(4);
```
Comment (behdad) [inline hbjs.js:537]: In theory you can do with one malloc?

Comment (khaledhosny) [reply hbjs.js:537]: How is that?

Comment (behdad) [reply hbjs.js:537]: I was thinking allocing 12 bytes and sending in three chunks of it somehow... Nevermind though.

Comment (khaledhosny) [reply hbjs.js:537]: I see. I don’t think it is a real malloc though, so probably it does not make much difference.

Comment (khaledhosny) [reply hbjs.js:537]: Something like this I guess:
```js
    var versionPtr = exports.malloc(4 * 3);
    var major = versionPtr;
    var minor = versionPtr + 4;
    var micro = versionPtr + 8;
```

Comment (khaledhosny) [reply hbjs.js:537]: https://github.com/harfbuzz/harfbuzzjs/commit/fb92648599d11635b656715b158d538a6cbbae7f

**Pull request merged**